apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-checkbox-styling.component.ts

```diff
@@ -0,0 +1,27 @@
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { Component, DoCheck } from '@angular/core';
+
+@Component({
+    selector: 'fdp-checkbox-style',
+    templateUrl: 'platform-checkbox-styling.component.html',
+})
+export class PlatformChekboxStyleComponent implements DoCheck {
+    customForm = new FormGroup({
+        example1: new FormControl({ value: '', disabled: false }),
+        example2: new FormControl({ value: '', disabled: false }, Validators.required),
+    });
+
+    ngDoCheck() {
+        if (this.customForm.get('example1').value !== true) {
```

Comment:
Store reference to object in varible to avoid unecessary method calls, like:
```let exampleRef = this.customForm.get('example1');

exampleRef.value !== true
...
```

Reply:
using reference now to avoid multiple calls.